docs/source/reference/math.rst

```diff
@@ -47,27 +49,27 @@ Rounding
    cupy.around
    cupy.round_
    cupy.rint
+   cupy.fix
    cupy.floor
    cupy.ceil
    cupy.trunc
-   cupy.fix
 
 
-Sums and products
------------------
+Sums, products, differences
```

Comment:
This is just a comment but cupy doesn't have routines for differences yet.

Reply:
OK, It is acceptable.